e2e-tests/cypress/e2e/journeys/insurance/your-business/turnover/validation/turnover-percentage-turnover-validation.spec.js

```diff
@@ -0,0 +1,202 @@
+        const field = turnover[fieldId];
+
+        cy.checkText(field.error(), `Error: ${errorMessage}`);
+      });
+    });
+
+    describe(`when ${PERCENTAGE_TURNOVER} is over 100`, () => {
+      const errorMessage = TURNOVER_ERRORS[PERCENTAGE_TURNOVER].ABOVE_MAXIMUM;
+
+      it(`should display validation errors for ${PERCENTAGE_TURNOVER}`, () => {
+        const fieldId = PERCENTAGE_TURNOVER;
+        const field = turnover[fieldId];
+
+        field.input().clear().type('120');
+        submitButton().click();
+        partials.errorSummaryListItems().should('have.length', 2);
+        cy.checkText(partials.errorSummaryListItems().eq(1), errorMessage);
+      });
+
+      it(`should display the validation error for ${PERCENTAGE_TURNOVER}`, () => {
+        const fieldId = PERCENTAGE_TURNOVER;
+        const field = turnover[fieldId];
+
+        cy.checkText(field.error(), `Error: ${errorMessage}`);
+      });
+    });
+
+    describe(`when ${PERCENTAGE_TURNOVER} is below 0`, () => {
+      const errorMessage = TURNOVER_ERRORS[PERCENTAGE_TURNOVER].BELOW_MINIMUM;
+
+      it(`should display validation errors for ${PERCENTAGE_TURNOVER}`, () => {
+        const fieldId = PERCENTAGE_TURNOVER;
+        const field = turnover[fieldId];
+
+        field.input().clear().type('-15');
+        submitButton().click();
+        partials.errorSummaryListItems().should('have.length', 2);
+        cy.checkText(partials.errorSummaryListItems().eq(1), errorMessage);
+      });
+
+      it(`should display the validation error for ${PERCENTAGE_TURNOVER}`, () => {
+        const fieldId = PERCENTAGE_TURNOVER;
+        const field = turnover[fieldId];
+
+        cy.checkText(field.error(), `Error: ${errorMessage}`);
+      });
+    });
+  });
+
+  describe(`when ${PERCENTAGE_TURNOVER} is correctly entered as a whole number`, () => {
+    it('should not display validation errors', () => {
+      cy.navigateToUrl(url);
+
+      const fieldId = PERCENTAGE_TURNOVER;
+      const field = turnover[fieldId];
+
+      field.input().clear().type('5');
+      submitButton().click();
+      partials.errorSummaryListItems().should('have.length', 1);
+    });
+  });
+
+  describe(`when ${PERCENTAGE_TURNOVER} is correctly entered with a comma`, () => {
+    it('should not display validation errors', () => {
+      cy.navigateToUrl(url);
+
+      const fieldId = PERCENTAGE_TURNOVER;
+      const field = turnover[fieldId];
+
+      field.input().clear().type('5,0');
+      submitButton().click();
+      partials.errorSummaryListItems().should('have.length', 1);
+    });
+  });
+
+  describe(`when ${PERCENTAGE_TURNOVER} is correctly entered as 0`, () => {
+    it('should not display validation errors', () => {
+      cy.navigateToUrl(url);
+
+      const fieldId = PERCENTAGE_TURNOVER;
+      const field = turnover[fieldId];
+
+      field.input().clear().type('0');
+      submitButton().click();
+      partials.errorSummaryListItems().should('have.length', 1);
+    });
+  });
```

Comment:
What do you think about using a shared assertion function? Each scenario/test is very repetitive, had the same with password validation: https://github.com/UK-Export-Finance/exip/blob/main-application/e2e-tests/cypress/e2e/journeys/insurance/account/create/your-details/account-create-your-details-validation-password.spec.js#L27

Could do this for kinds of test for "your business" after this PR as a tech improvement